src/test/java/org/homeschoolpebt/app/cli/TransmitterCommandsTest.java

```diff
@@ -132,7 +151,7 @@ void transmitZipFile() throws IOException, JSchException, SftpException {
     DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy-MM-dd");
     LocalDateTime now = LocalDateTime.now();
     String date = dtf.format(now);
-    File zipFile = new File("Apps__" + date + "__1001-1003.zip");
+    File zipFile = new File("Apps__" + date + "__1001-1004.zip");
```

Comment:
Interesting that the submission that's being held for docs is included in the filename range despite not actually being in the range. Not ideal but also not a deal breaker.